fem/fe.cpp

```diff
@@ -466,6 +466,24 @@ void PositiveFiniteElement::Project(
    }
 }
 
+void PositiveFiniteElement::Project(
+   VectorCoefficient &vc, ElementTransformation &Trans, Vector &dofs) const
+{
+   MFEM_ASSERT(dofs.Size() == vc.GetVDim()*Dof, "");
+   Vector x(vc.GetVDim());
+
+   for (int i = 0; i < Dof; i++)
+   {
+      const IntegrationPoint &ip = Nodes.IntPoint(i);
+      Trans.SetIntPoint(&ip);
+      vc.Eval (x, Trans, ip);
+      for (int j = 0; j < x.Size(); j++)
+      {
+         dofs(Dof*j+i) = x(j);
```

Comment:
Support both orderings?

Reply:
This method was copied almost directly from `NodalFiniteElement::Project` so it uses the same ordering as that class.  It was my understanding that element-level classes all use this ordering and the `FiniteElementSpace` is responsible for reordering during assembly if necessary.

Reply:
You are right. Thanks Mark!